README.md

```diff
@@ -7,10 +7,98 @@ new [cisagov](https://github.com/cisagov) Go project started.
 This skeleton project contains [licensing information](LICENSE), as
 well as [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+appropriate for a Go package.
 
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
+## Installation ##
```

Comment:
Please update to include an explanation for how to build with a specific version in the generated binary.

Reply:
> Please update to include an explanation for how to build with a specific version in the generated binary.

what do you mean? can you please explain? I am under the impression that if a user were looking for a specific version they would just clone and build from that?

Reply:
Apologies for being unclear. I mean a short blurb that explains how to build a binary such that https://github.com/cisagov/skeleton-golang-package/blob/56dc6d7a71418499be8dce1cdce7c056b00c943e/main.go#L40 is populated with an actual version in the binary.

Reply:
@mcdonnnj aah, gotcha. Your comment made me realize maybe it's better to not hard-code the version number but grab it from git instead. So I've made some mods. Please see https://github.com/cisagov/skeleton-golang-package/commit/e58b056aa822248cfc1ed659ab56df0384636e3c

Reply:
The `(devel)` is only supposed to be a placeholder so that there is an actual version value no matter how the binary is built. The value is overwritten when you build per your example so there should be no harm in leaving the value as it matches the default module version you would get from a [BuildInfo struct](https://pkg.go.dev/runtime/debug#BuildInfo).

Reply:
Sure, please see https://github.com/cisagov/skeleton-golang-package/pull/2/commits/470bf50080d356d23a104dbbba4c9395a1b72e47